### DISCUSSION TITLE
Recommendations for embedding videos (e.g. from Youtube)?

### DISCUSSION THREAD

**Discussion by nktnet1** [fuma-nama/fumadocs#792] [Q&A] [answered]
I really like this image-zoom component:
- https://fumadocs.vercel.app/docs/ui/components/image-zoom

for its animations and responsiveness.

Do you have any recommendations for embedding videos (specifically from YouTube) into fumadocs?

Thank you!

**Comment by fuma-nama** [comment, marked as answer]
React YouTube is great enough

**Reply by nktnet1** [reply]
I looked into [react-youtube](https://github.com/tjallingt/react-youtube), but ultimately stuck with iframes and using [tailwind aspect-ratio classes](https://tailwindcss.com/docs/aspect-ratio). E.g.

```tsx
<iframe
  src="https://www.youtube-nocookie.com/embed/<SOME ID>"
  className="w-full aspect-video"
  allowFullScreen
/>
```

Thank you for replying regardless though!
